Code/CSharpCollections1/Task_01.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.Design;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpCollections1
+{
+    public class Task_01
+    {          
+        public void SortArray(int[] array) 
+        {
+            Array.Sort(array);            
+        }
+        
+        public int[] ParseArray(string input)
+        {         
+            string[] numbers = input.Split(' ');
+
+            int[] array = new int[numbers.Length];
+            for (int i = 0; i < numbers.Length; i++)
+            {
+                if (!int.TryParse(numbers[i], out array[i]))
+                {
+                    Console.WriteLine($"Error! Invalid input: {input}");
+                    return new int[0];
```

Comment:
Do you really need to return value from massive here?

Reply:
In the provided code, the statement "return new int[0];" is used to handle the case where the input string cannot be parsed into an integer array. If the input string contains invalid or non-numeric characters, the parsing will fail for at least one element of the array. In this case,  to handle the error condition by returning an empty array.
By returning "new int[0]", the code indicates that an error has occurred during the parsing process. The calling code can check the returned array's length to determine if the parsing was successful or not. If the length is zero, it implies that there was an error, and the calling code can handle the error condition accordingly.
Returning an empty array in error conditions is just one way to handle errors. It is also possible to use throwing an exception in this case but I completed this before the "Exception" lesson.

Reply:
Good, I got it? thanks!